pkg/gtk4-layer-shell/build.zig.zon

```diff
@@ -0,0 +1,15 @@
+.{
+    .name = "gtk4-layer-shell",
+    .version = "0.0.0",
```

Comment:
We try to keep this matched up to the dependency we build, so this should also be 1.1.0.